apps/fabric-website/index.html

```diff
@@ -184,8 +184,9 @@
       }
 
       function loadAppScripts(appPath) {
-        var reactPath = '//cdnjs.cloudflare.com/ajax/libs/react/16.8.3/umd/react.production.min.js';
-        var reactDomPath = '//cdnjs.cloudflare.com/ajax/libs/react-dom/16.8.3/umd/react-dom.production.min.js';
+        var version = isLocal ? '.development.js' : '.production.min.js';
```

Comment:
@kenotron I think we should bring back the development build of React for local testing--without it, debugging any issues which happen to occur within React's render loop becomes much more difficult